### DISCUSSION TITLE
Car charger entity in the energy dashboard

### DISCUSSION THREAD

**Discussion by robinostlund** [home-assistant/frontend#12992] [Dashboard]
Hi,
It would be nice if it where possible to add a car charger with a kwh sensor to the energy dashboard. Maby to be future ready maby in both directions as it is coming cars with V2X support.

**Comment by KTibow** [comment]
Could you elaborate? You can already add [individual devices](https://www.home-assistant.io/docs/energy/individual-devices/) to the energy dashboard.

**Reply by robinostlund** [reply]
Yeah that i know but something visual :) ferroamp has a dashboard like this:
https://capture.dropbox.com/AzA26tUqzaE3BWd2

**Reply by robinostlund** [reply]
And as v2x is coming both incomming and outgoing energy will be possible later on.
